src/openforms/config/models.py

```diff
@@ -661,14 +662,25 @@ def as_dict(self):
 class CSPSettingManager(models.Manager.from_queryset(CSPSettingQuerySet)):
     @transaction.atomic
     def set_for(
-        self, obj: models.Model, settings: list[tuple[CSPDirective, str]]
+        self,
+        obj: models.Model,
+        *settings: tuple[CSPDirective, str] | tuple[CSPDirective, str, str],
```

Comment:
Makes the annotation a bit simpler, but can be confusing to use. So I can revert if prefered.